lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb

```diff
@@ -37,14 +38,18 @@ def transform(data)
           rated_disability_id: 0,
           rating_decision_id: 0,
           rating_percentage: rated_disability['rating_percentage'],
+          # TODO: figure out if this is important
```

Comment:
There may be some work here, but we don't know yet because we haven't seen the Lighthouse API call work in higher environments yet.

On our local environment, we saw the UI behave in the expected way, but I wanted to note this here in case there is something failing, we have something to look at first.